tutorials/proto_rename/person.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package com.google.startupos.tutorials.proto_rename;
+
+option java_package = "com.google.startupos.tutorials.proto_rename";
+option java_outer_classname = "Protos";
+
+
+message Person {
+    string name = 1;
+    int32 id = 2;
+
+    enum PhoneType {
```

Comment:
Let's simplify - instead of `PhoneNumber`, `repeated PhoneNumber phone` and `enum PhoneType`, let's have:
`enum FAVORITE_PIZZA_TOPPING` with mushrooms, olives and pineapple, and add a field for that.

Reply:
Done.